packages/storybook-vue/stories/components/dropdown-select/ScaleDropdownSelect.vue

```diff
@@ -36,6 +40,10 @@ export default {
     hideLabelVisually: { type: Boolean, default: false },
     floatingStrategy: { type: String },
     ariaDetailsId: { type: String },
+    readOnly: { type: Boolean }, 
+    transparent: { type: Boolean }, 
+    ariaLabelSelected: { type: String },
+    hcmLabelDisabled: { type: String },
```

Comment:
we can also add the description for these attributes here : 
e.g.  for `hcmLabelDisabled`
https://github.com/telekom/scale/blob/main/packages/components/src/components/dropdown-select/readme.md?plain=1#L17